python/src/RandomVectorMetropolisHastings_doc.i.in

```diff
@@ -31,33 +38,37 @@ Examples
 
 Sample from a random variable:
 
->>> source = ot.RandomVector(ot.Normal())
+>>> randomVector = ot.RandomVector(ot.Normal())
 >>> initialState = [3.0]
->>> sampler = ot.RandomVectorMetropolisHastings(source, initialState)
->>> sampler.setBurnIn(20)
->>> sampler.setThinning(2)
+>>> sampler = ot.RandomVectorMetropolisHastings(randomVector, initialState)
 >>> x = sampler.getSample(10)
 
 We can also link the parameters of the random vector to the state of the chain,
 here the parameters of the random vector being the ones of the underlying distribution:
 
->>> slf = ot.SymbolicFunction(['x'], ['0.0', '0.1'])
->>> sampler = ot.RandomVectorMetropolisHastings(source, initialState, [0], slf)"
+>>> linkFunction = ot.SymbolicFunction(['x'], ['x', '0.1'])
+
+The link function makes the first parameter of the normal distribution (the mean)
+equal to the current value of the Markov chain.
+Its standard deviation remains constant: :math:`0.1`.
+That way we construct a random walk with normal steps of standard deviation :math:`0.1`.
+
+>>> sampler = ot.RandomVectorMetropolisHastings(randomVector, initialState, [0], linkFunction)"
 
 // ---------------------------------------------------------------------
 
-%feature("docstring") OT::RandomVectorMetropolisHastings::getSource
+%feature("docstring") OT::RandomVectorMetropolisHastings::getRandomVector
 "Get the proposal distribution.
 
 Returns
 -------
-source : :class:`~openturns.RandomVector`
+randomVector : :class:`~openturns.RandomVector`
     The distribution from which the transition kernels of the random walk
     Metropolis-Hastings algorithm is defined"
 
 // ---------------------------------------------------------------------
 
-%feature("docstring") OT::RandomVectorMetropolisHastings::setSource
+%feature("docstring") OT::RandomVectorMetropolisHastings::setRandomVector
 "Set the proposal distribution.
```

Comment:
```suggestion
"Set the random vector.
```

Reply:
Note changes also need to be made to the parameter description immediately below this line, but GitHub wouldn't let me reach those lines.